sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -361,6 +400,37 @@ const reclaimAssetInHTLC = async (
     return result;
 };
 
+/**
+ * Rollback step of a Hashed Time Lock Contract
+ * - Reclaim a unique asset instance using contractId
+ **/
+const reclaimAssetInHTLCusingContractId = async (
+    contract: Contract,
+    contractId: string,
+    recipientECert: string,
```

Comment:
I think recipientEcert can be removed, it is not being used and I guess not required as well in reclaim.

Reply:
Done @sanvenDev !